docs/StardustDocs/topics/extensionPropertiesApi.md

```diff
@@ -28,12 +39,9 @@ df.add("lastName") { name.split(",").last() }
 
 The `titanic.csv` file could be found [here](https://github.com/Kotlin/dataframe/blob/master/data/titanic.csv).
 
-In notebooks, extension properties are generated for [`DataSchema`](schemas.md) that is extracted from [`DataFrame`](DataFrame.md) 
-instance after REPL line execution. 
-After that [`DataFrame`](DataFrame.md)  variable is typed with its own [`DataSchema`](schemas.md), so only valid extension properties corresponding to actual columns in DataFrame will be allowed by the compiler and suggested by completion.
-
 Extension properties can be generated in IntelliJ IDEA using the [Kotlin Dataframe Gradle plugin](schemasGradle.md#configuration).
 
 <warning>
-In notebooks generated properties won't appear and be updated until the cell has been executed. It often means that you have to introduce new variable frequently to sync extension properties with actual schema
+In notebooks generated properties won't appear and be updated until the cell has been executed.
+It often means that you have to introduce new variable frequently to sync extension properties with actual schema.
```

Comment:
NOTE: ?

Reply:
No need, the warning tags already make it look like this:
![image](https://github.com/Kotlin/dataframe/assets/17594275/d77fc0eb-bbd8-40d0-a359-0215170fb141)